tests/tree.rs

```diff
@@ -1693,6 +1693,16 @@ fn basic() {
     assert_eq!(r.value, &0);
     assert_eq!(r.params(), vec![]);
 
+    tree.insert("", 14);
+    let r = tree.find("/").unwrap();
+    assert_eq!(r.value, &14);
```

Comment:
This is a bit confusing. The path goes in as `"/"` but comes out as `"/"`?

Reply:
`tree.insert("", 14)` allow change the start node value , but it does not modify the node prefix path.

1. `tree.insert("", 14)` node path = ''
2. `tree.insert("/", 15)`  node path = '/', overwrite the empty string

Reply:
Or do not allow empty string, users handle their own paths

Reply:
here is another way to allow '' to be the root node of the tree and '/' to be its child node. (@adriangb  This should be what you expect).

Reply:
Yea I would expect `“/“` to be a child of `””`

Reply:
Because, I always consider that the real request path does not appear `''`, when organizing the routing system, it has to pre-process path at the users

Reply:
For what it's worth I was easily able to work around this one level of abstraction up: https://github.com/adriangb/routrie/commit/70fc7c52d6da646f1bb4262acf6fb689189caebc

I think the fact that this can be solved one level of abstraction up indicates that the limitation is artificial/due to the implementation, not a conceptual one.